install-panel.sh

```diff
@@ -711,49 +682,131 @@ function main {
       printf '*'
     fi
   done
-
   if [ -z "$MYSQL_PASSWORD" ]; then
     print_error "MySQL password cannot be empty"
     exit 1
   fi
-
   print_brake 72
 
   # set FQDN
-
-  echo -n "* Set the FQDN of this panel (panel hostname): "
+  echo -n "* Set the FQDN of this panel (panel.example.com): "
   read -r FQDN
-
+  
   echo ""
-
-  echo "* This installer does not configure Let's Encrypt, but depending on if you're"
-  echo "* going to use SSL or not, we need to know which webserver configuration to use."
-  echo "* If you're unsure, use (no). "
-  echo -n "* Assume SSL or not? (yes/no): "
+  echo "* We need to know which webserver configuration to use."
+  echo "* If you're unsure, press Enter. "
+  echo -n "* Assume SSL or not? (y/N): "
   read -r ASSUME_SSL_INPUT
-
-  if [ "$ASSUME_SSL_INPUT" == "yes" ]; then
+  if [ "$ASSUME_SSL_INPUT" == "y" ]; then
     ASSUME_SSL=true
-  elif [ "$ASSUME_SSL_INPUT" == "no" ]; then
+  elif [ "$ASSUME_SSL_INPUT" == "Y" ]; then
+    ASSUME_SSL=false
+  elif [ "$ASSUME_SSL_INPUT" == "n" ]; then
+    ASSUME_SSL=false
+  elif [ "$ASSUME_SSL_INPUT" == "N" ]; then
     ASSUME_SSL=false
   else
-    print_error "Invalid answer. Value set to no."
+    print_error "Default answer. Value set to no."
     ASSUME_SSL=false
   fi
 
   # confirm installation
-  echo -e -n "\n* Initial configuration completed. Continue with installation? (y/n): "
+  echo -e -n "\n* Initial configuration completed. Continue with installation? (y/N): "
   read -r CONFIRM
   if [ "$CONFIRM" == "y" ]; then
     perform_install
+  elif [ "$CONFIRM" == "Y" ]; then
+    exit 0
   elif [ "$CONFIRM" == "n" ]; then
     exit 0
+  elif [ "$CONFIRM" == "N" ]; then
+    exit 0
   else
     # run welcome script again
-    print_error "Invalid confirm. Will exit."
+    print_error "Invalid confirm. Will repeat."
     exit 1
   fi
+}
+
+function ufw-Ubuntu {
+  #This function is to use UFW on Ubuntu to Open ports based on whether the user is going
+  #to be hosting the daemon on their machine or just the panel.
+  #When you want to call the total UFW function, you call this function
+
+  read -p "Are you going to be installing the Daemon on this machine (y/N): " DaemonAsk
```

Comment:
Perhaps this could say "Are you going to install the Pterodactyl daemon on this machine?"

Also, all questions should be asked within the `main` function and not here.